platform/ios/src/MGLAnnotationView.h

```diff
@@ -13,7 +13,7 @@ NS_ASSUME_NONNULL_BEGIN
  @param reuseIdentifier The string that identifies that this annotation view is reusable.
  @return The initialized annotation view object or `nil` if there was a problem initializing the object.
```

Comment:
The return type isn’t marked `nullable`. I realize the MKAnnotationView documentation also indicates that `nil` can be returned, but I think that’s a holdover from the old days before nullability was a concept in Objective-C. Is this view that much more likely to fail to initialize than an ordinary UIView?